routes/articles.js

```diff
@@ -10,6 +10,20 @@ router.get('/', (req, res) => {
     })
 })
 
+
+router.post('/article', (req,res) => {
+    const db = req.app.get('db')
+    const title  = req.body.title
+    const body = req.body.body
+    const author = req.body.author
+    
+    db.query('INSERT INTO articles(title,body,author) VALUES(?,?,?)',[title,body,author], (err) => {
+        if(err) console.log(err)
+        else res.status(200)
```

Comment:
This is not needed. Response status is 200 by default.
```suggestion
```

Reply:
fixed